### ISSUE TITLE
Accessibility - Main Country List Screen - Missing Top of Screen Heading (Title)

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#31]
Missing heading at top of the screen telling a person using TalkBack what screen they are on. 

The problem is for all the different tabs on this screen (A –Z, By Level, and By Region).

**Comment by aquilent-dgemoets** [comment]
See discussion for #20.  We will try to add an alt tag.

**Comment by jfc3-dol** [comment]
I reviewed the ILAB 'Sweat and Toil' application on OPA's 'Galaxy Tab S' and an ILAB employees 'LG Tribute 2' and both did not read any headings. 

From my understanding two of the 'More information' screens (About the App and Methodology) where HTML pages and had the headings in them. 

I thought the OSHA 'Heat Index' application had similar screens and the headings with their proper level were read for HTML screens before. The other Android screens that had what equated to headings were read too.

**Comment by aquilent-dgemoets** [comment]
I checked the repository for the (unpublished) Android version of the OSHA 'Heat Index' application and didn't see any added accessibility code/configuration.  Could you confirm that the behavior you are remembering was for the Android version of the app?  If so, maybe we can find out how they did it.

**Comment by jfc3-dol** [comment]
Looks  like Android does not read the top banner/header in same way as iOS, which had the text up there as a heading. 

Closing issues.
